app/Document.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Document extends Model {
+
+
+public function Authors(){
+	return $this->belongsTo(Author::class);
+}
+
+public function Files(){
```

Comment:
Same as above.

Reply:
Are you sure this method is actually needed? Or you just put them here thinking in the next functionalities?